lib/id_generator/configuration.rb

```diff
@@ -0,0 +1,24 @@
+module IdGenerator
+  class Configuration
+    attr_reader :context_id
+
+    def initialize(context_id: 0)
```

Comment:
Are you sure you want to default it here?

Reply:
@Fattaf Let's leave it for now. My thought process was that configuration and defaults leave in one place. I agree that if we have more - we can move it somewhere